clients/src/main/java/org/apache/kafka/common/protocol/SecurityProtocol.java

```diff
@@ -26,6 +26,8 @@
     PLAINTEXT(0, "PLAINTEXT"),
     /** SSL channel */
     SSL(1, "SSL"),
+    /** SASL authenticated, non-encrypted channel */
+    PLAINTEXTSASL(2, "PLAINTEXTSASL"),
```

Comment:
OK, thanks. Should the name of this be `SASL_PLAINTEXT` and `SSLSASL` should be `SASL_SSL`? I guess this is a bit subjective, but seems a bit clearer to me.